### PR TITLE
Docs/client cert security

### PR DESCRIPTION
**Pull request by gra-moore**

### CODE REVIEW

**Review by ashmanto — APPROVED** [review]

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -1292,15 +1292,46 @@ You can tune the LOG_LEVEL of the Datahub. The supported values are DEBUG and IN
 
 The Datahub supports reporting metrics trough a StatsD server. This is turned off if left empty, and you can turn it on by giving it an ip-address and a port combination.
 
-#### Securing the Datahub
+#### Securing the Data Hub
+
+There are four main security models for the data hub. 
+
+1. No security / API gateway seured. All calls are allowed at the data hub API level. This mode can be used either when developing or when the data hub API is protected behind an API gateway that implements secure access.
```
Comment (4hlberg) [inline DOCUMENTATION.md:1299]: secured

File: DOCUMENTATION.md [hunk 2]
```diff
@@ -1292,15 +1292,46 @@ You can tune the LOG_LEVEL of the Datahub. The supported values are DEBUG and IN
...
 The Datahub supports reporting metrics trough a StatsD server. This is turned off if left empty, and you can turn it on by giving it an ip-address and a port combination.
 
-#### Securing the Datahub
+#### Securing the Data Hub
+
+There are four main security models for the data hub. 
+
+1. No security / API gateway seured. All calls are allowed at the data hub API level. This mode can be used either when developing or when the data hub API is protected behind an API gateway that implements secure access. 
+
+2. Data Hub Security. This involves a datahub allowing for the registration of clients and a public key. The client (often in this model another datahub) retrieves a JWT access token by sending a request (signed with a private key) to authenticate.
```
Comment (4hlberg) [inline DOCUMENTATION.md:1301]: "allowing the registration" sounds better?

File: DOCUMENTATION.md (hunk 1 above)
Comment (gra-moore) [reply DOCUMENTATION.md:1299]: secure is correct.

File: DOCUMENTATION.md (hunk 2 above)
Comment (gra-moore) [reply DOCUMENTATION.md:1301]: 'allowing for' was intended. Both are ok.

**Review by 4hlberg — APPROVED** [review]

**Pull request merged**